py_wtf/cli/__init__.py

```diff
@@ -64,6 +70,42 @@ async def index(project_name: str, directory: str, pretty: bool, force: bool) ->
         rich.print(proj)
 
 
+@py_wtf.command()
+@click.argument("directory")
+@click.option("--top", type=int, default=50)
+@coroutine
+async def index_top_pypi(directory: str, top: int) -> None:
+    out_dir = Path(directory)
+    out_dir.mkdir(parents=True, exist_ok=True)
+    repo = ProjectRepository(out_dir)
+    async with httpx.AsyncClient() as client:
+        top_pkgs = (
+            await client.get(
+                "https://hugovk.github.io/top-pypi-packages/top-pypi-packages-30-days.json"
+            )
+        ).json()
+        projects: Iterable[str] = (row["project"] for row in top_pkgs["rows"][:top])
```

Comment:
@abesto lookie here

Reply:
Ah! How... did I miss that. Oh well.

Reply:
... swear this looked different when I last checked.